ci/tasks/parse-release-tag.yml

```diff
@@ -0,0 +1,16 @@
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+inputs:
+  - name: git-release
+outputs:
+  - name: tags
+run:
+  path: sh
+  args:
+    - -ec
+    - |
+      RELEASE_NUMBER=$(cat git-release/.git/ref | sed 's/alpha_release-//')
+      echo "${RELEASE_NUMBER}-release" > tags
```

Comment:
does this need to be `> tags/tag` ? I'm looking at this thinking that `tags` is the output directory and it needs to write into a file within that directory?

Reply:
yep. KInda forgot how outputs actually work